3rdparty/ittnotify/src/ittnotify/jitprofiling.c

```diff
@@ -61,9 +61,6 @@
 #if ITT_PLATFORM==ITT_PLATFORM_WIN
 #include <windows.h>
 #endif /* ITT_PLATFORM==ITT_PLATFORM_WIN */
-#if ITT_PLATFORM != ITT_PLATFORM_MAC && ITT_PLATFORM != ITT_PLATFORM_FREEBSD
-#include <malloc.h>
-#endif
```

Comment:
Why do you want to break non-BSD platforms?

---

3rdparty patches should go to their upstream repositories first.

Reply:
>Why do you want to break non-BSD platforms?

I don't, of course.  Why do you think this will?  Did you see the details in my commit message?

>3rdparty patches should go to their upstream repositories first.

I suspected so, which is why I kept that one in a separate commit. Do you know where upstream is for that library?  I'll drop that commit.

Reply:
The commit to 3rd party is now dropped.

Reply:
@opencv-alalek with the change to 3rd party removed, can this be merged now?

Reply:
I found upstream, and there is already an issue for this: https://github.com/intel/ittapi/issues/65